src/core/src/bootstrap/Constants.py

```diff
@@ -126,6 +126,11 @@ class AutoAssessmentStates(EnumBackport):
         DISABLED = "Disabled"
         ENABLED = "Enabled"
 
+    # To separately preserve assessment + auto-assessment state information
+    ASSESSMENT_STATE_FILE = "AssessmentState.json"
+    AUTO_ASSESSMENT_MAXIMUM_DURATION = "PT1H"
+    MIN_AUTO_ASSESSMENT_INTERVAL = "PT6H"   # do not perform auto-assessment if the last assessment happened less than this time interval ago
```

Comment:
Are these used in the code anywhere?

Reply:
Not yet. Will be used in a future PR. Removed for now.